src/pages/userPage/Approvers/Approvers.tsx

```diff
@@ -29,18 +31,33 @@ const Assignments = () => {
   const [data, setData] = useState<ApproversData>();
   const [approverName, setApproverName] = useState<string>();
   const [userGender, setuserGender] = useState<string>();
+  const [accessLevels, setAccessLevels] = useState<string[]>([]);
   const userGenders = ["Чоловік", "Жінка", "Інша"];
+  const AccessableRoles=["Admin", "Голова Куреня", "Голова Станиці", "Голова Округи", "Дійсний член організації", "Прихильник", "Зареєстрований користувач"];
+  const [roles, setRoles]=useState<string[]>([]);
 
   const fetchData = async () => {
     const token = AuthStore.getToken() as string;
     const user: any = jwt(token);
+      let decodedJwt = jwt_decode(AuthStore.getToken() as string) as any;
+      setRoles([].concat(decodedJwt[
+          "http://schemas.microsoft.com/ws/2008/06/identity/claims/role"
+          ]));
     await userApi.getApprovers(userId, user.nameid).then(response => {
       setData(response.data);
       setLoading(true);
     }).catch(() => { notificationLogic('error', fileIsNotUpload("даних")) });
+    setAccessLevels(await activeMembershipApi.getAccessLevelById(userId));
     fetchApproverName(user.nameid);
 };
 
+const AccessToManage=(roles: string[]):boolean=>{ 
+    for(var i = 0; i < roles.length; i++){
+       if(AccessableRoles.includes(roles[i])) return true;
```

Comment:
Try something just like that:

```suggestion
      return AccessableRoles.includes(roles[i]);
```

If it will include this role, it should return **true**, otherwise - **false**